qrencode.el

```diff
@@ -1084,6 +1084,15 @@ Commands:
         (message "No URL found at point")
       (qrencode--encode-to-buffer url))))
 
+;;;###autoload
+(defun qrencode-url (url)
+  "Encode URL.
+When called interacively, read URL with `read-string'."
+  (interactive `(,(read-string "URL: ")))
+  (if (null (string-trim url))
```

Comment:
`(null "")` returns `nil`. Maybe you mean `string-empty-p`?

Reply:
Done.